plugin/kubernetes/controller.go

```diff
@@ -431,6 +431,67 @@ func (dns *dnsControl) updateModifed() {
 	atomic.StoreInt64(&dns.modified, unix)
 }
 
-func (dns *dnsControl) Add(obj interface{})               { dns.updateModifed() }
-func (dns *dnsControl) Delete(obj interface{})            { dns.updateModifed() }
-func (dns *dnsControl) Update(objOld, newObj interface{}) { dns.updateModifed() }
+func (dns *dnsControl) Add(obj interface{})    { dns.updateModifed() }
+func (dns *dnsControl) Delete(obj interface{}) { dns.updateModifed() }
+
+func (dns *dnsControl) Update(objOld, newObj interface{}) {
+	// endpoint updates can come frequently, make sure
+	// it's a change we care about
+	if o, ok := objOld.(*api.Endpoints); ok {
+		n := newObj.(*api.Endpoints)
+		if endpointsEquivalent(o, n) {
+			return
+		}
+	}
+	dns.updateModifed()
+}
+
+// checks if the update to an endpoint is something
```

Comment:
It's not exported, but it still good form to start the comment with `endpointsEquivalent` (just endpontsEqual to mimic std lib naming?)

Reply:
Ok. They are not equal in the normal sense. They just produce the same DNS entries.